src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -229,6 +231,11 @@ class MediaBaseViewer extends BaseViewer {
             return;
         }
 
+        if (this.loaded) {
+            this.play(this.currentTime);
+            return;
+        }
+
```

Comment:
Does this need to make its way into `DashViewer`'s `loadeddataHandler` as well?

Reply:
`DashViewer` loads data in another way. Calling `this.player.retryStreaming()` is enough for `DashViewer`.